deploy/docs/Best_Practices.md

```diff
@@ -1158,15 +1151,15 @@ to work with our collection.
 
    - add your custom prometheus configuration
 
-1. [Upgrade sumologic chart](./Installation_with_Helm.md#upgrading-sumo-logic-collection) without `kube-prometheus-stack`
+2. [Upgrade sumologic chart](./Installation_with_Helm.md#upgrading-sumo-logic-collection) without `kube-prometheus-stack`
```

Comment:
tbh, markdownlint should complain about it :confused:

Reply:
There's a parameter for it: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md029---ordered-list-item-prefix but I think the default is permissive.